src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -133,5 +133,11 @@ public void should_return_a_200_and_when_performing_search_when_caseId_correct_e
             .contentType(APPLICATION_JSON_VALUE)
             .body("tasks.size()", equalTo(0));
     }
+
+    @After
+    public void cleanUp() {
+        camundaApiActions.post(ENDPOINT_COMPLETE_TASK, taskId,
+                               new Headers(authorizationHeadersProvider.getServiceAuthorizationHeader()));
+    }
```

Comment:
Again more of personal opinion but would it be better to move this to Common class inside the testUtils. mainly because this forces you to define ENDPOINT_COMPLETE_TASK in this class which is not really relevant to what the class is testing. 

That way we could have something like.

```
common.cleanUpTask(taskId)
```
or
```
common.tearDownTask(taskId)
```